src/day-1-toy/comp.py

```diff
@@ -16,17 +16,17 @@
 
 a = ["foo", "bar", "baz"]
 
-y = []
+y = [s.upper() for s in a]
 
 print(y)
 
 # Use a list comprehension to create a list containing only the _even_ elements
 # the user entered into list x.
 
 x = input("Enter comma-separated numbers: ").split(',')
-
+print(x)
 # What do you need between the square brackets to make it work?
-y = []
+y = [i for i in x if int(i)%2==0]
```

Comment:
This will output the numbers as strings. 

Compare to:  y = [int(i) for i in x if int(i) % 2==0]